src/app.js

```diff
@@ -22,7 +23,25 @@ client.on(Events.InteractionCreate, async interaction => {
   }
 
   try {
-    // Run the command, passing along the interaction
+    // Run the command, passing along the interaction 
+    if (interaction.commandName == 'deposit') {
+      const userId = interaction.user.id;
+      const now = Date.now();
+      const cooldownAmount = 3 * 60 * 1000;
```

Comment:
This is 3 minutes, right? Can you add a comment like "// 3 minutes" for clarity?